test/integration/test_http.py

```diff
@@ -194,5 +194,44 @@ def test_read_write_txn(self):
         status = txn2.end()
         self.assertEqual(status.code, 200)
 
+    def test_schema(self):
```

Comment:
We don't need to be exhaustive in the testing here since the create schema logic is tested in the C++ tests, but we should test at least a couple failure cases here. Maybe a schema without any key fields specified, a schema with the same field name repeated, and trying to get a schema that doesn't exist.

Reply:
Added more tests.